xapian-applications/omega/omegatest

```diff
@@ -205,14 +205,13 @@ testcase '6' P=text a=fish b="Hello fish"
 testcase '' P=text a="Example" b="random"
 
 # Feature tests for $match.
-printf '$match{$cgi{a},$cgi{b}}' > "$TEST_TEMPLATE"
-testcase 'true' P=text a="abcde" b="abcde"
+printf '$match{$cgi{a},$cgi{b},$cgi{c}}' > "$TEST_TEMPLATE"
 testcase 'true' P=text a="ab*bc" b="abbbbbbc"
 testcase '' P=text a="a(^bc)d" b="azd"
-testcase 'true' P=text a="(.*)c(.*)" b="abcde"
-testcase 'true' P=text a="(bc+d$|ef*g.|h?i(j|k))" b="bcccd"
-testcase 'true' P=text a="(?<!\\?)'(.*?)(?<!\\?)'" b="'ab?'cd'efg"
-testcase 'true' P=text a="^([ab]*?)(?!(b))c" b="ababc"
+testcase 'true' P=text a="abc.*" b="ABCab\n" c="i" c="m" 
```

Comment:
This isn't how `$cgi{}` works; it will pick only one of those "arbitrarily". You actually want `c="im"`. You also aren't actually testing the `m` option here; you probably want something like:

```
testcase 'true' P=text a="abc$" b="ABC\nAB" c="im"
```

to test that.